mqtt/mqtt-edgehub/src/connection/delivery.rs

```diff
@@ -0,0 +1,137 @@
+use std::{collections::HashMap, sync::Arc};
+
+use async_trait::async_trait;
+use lazy_static::lazy_static;
+use mqtt_broker::{
```

Comment:
nit: rearrange

Reply:
It'd be nice to have a plugin to formatter for custom formatting rules